tests/composite_pk/models/tenant.py

```diff
@@ -14,17 +14,18 @@ class Token(models.Model):
     secret = models.CharField(max_length=10, default="", blank=True)
 
 
-class BaseModel(models.Model):
+class AbstractUser(models.Model):
     pk = models.CompositePrimaryKey("tenant_id", "id")
     tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE)
+    email = models.EmailField(unique=True)
     id = models.SmallIntegerField(unique=True)
 
     class Meta:
         abstract = True
 
 
-class User(BaseModel):
-    email = models.EmailField(unique=True)
+class User(AbstractUser):
+    pass
```

Comment:
I figured this was the easier way to keep testing abstract inheritance while mixing field order.